### ISSUE TITLE
PHP Warning: preg_replace_callback(): Unknown modifier 'P' in ParserService.php line 382

### ISSUE THREAD

**Issue by cedricziel** [featdd/dpn_glossary#74]
```
PHP 7.0.11 (cli) (built: Oct 11 2016 10:23:48) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with the ionCube PHP Loader (enabled) + Intrusion Protection from ioncube24.com (unconfigured) v6.0.4, Copyright (c) 2002-2016, by ionCube Ltd.
```

**Comment by featdd** [comment]
when is the exception thrown? could you send the specific markup so i can test it later?

**Comment by cedricziel** [comment]
I have no clue and no backtrace - sorry. I encountered it when checking the logs of a system that went through a migration.

**Comment by featdd** [comment]
Is the error still present?

**Comment by cedricziel** [comment]
It spams the system log continously, so I'd think it is.

**Comment by featdd** [comment]
hm can be a problem using / as a delimiter in my regex...
have to try that at home or maybe you can, just replace the "/" with a "#"

**Comment by featdd** [comment]
fixed in e87eb34a82440405dc5f80fe44ec1e324717b187
